src/qtiCommonRenderer/renderers/interactions/MediaInteraction.js

```diff
@@ -209,21 +202,29 @@ var _getRawResponse = function _getRawResponse(interaction) {
  *
  * Special value: the empty object value {} resets the interaction responses
  *
- * @param {object} interaction
- * @param {object} response
+ * @param {Object} interaction
+ * @param {Object} response
  */
-var setResponse = function(interaction, response) {
-    var responseValues;
+function setResponse(interaction, response) {
     if (response) {
         try {
-            //try to unserialize the pci response
-            responseValues = pciResponse.unserialize(response, interaction);
-            containerHelper.get(interaction).data('timesPlayed', responseValues[0]);
+            const maxPlays = parseInt(interaction.attr('maxPlays'), 10) || 0;
+            const responseValues = pciResponse.unserialize(response, interaction);
+            const timesPlayed = responseValues[0];
```

Comment:
I would move the parseInt call here, because in this case you would set the same data that you use in the condition.

Reply:
Make sense. I didn't want to change too much the stored data signature, but I don't think it will hurt.